src/sass/patterns/_animation.scss

```diff
@@ -0,0 +1,12 @@
+// Animation Patterns
+// ------------------
+
+@keyframes grow-in {
```

Comment:
I am growing the copied icon in, but since it doesn't exist after the timer I can't animate it out. I think it would be weird to animate all the copy icons in on page load. It doesn't really bother me that these switch as it might bring a bit more attention to the action you just took.

Reply:
I think the way this animation works looks good.

Reply:
@stacyk It would be possible to have it exist longer, if that would be helpful. Or maybe that's part of the story where the arrow morphs into the checkmark?

Reply:
@jamesnw  I don't think having it there longer would fix it as the issue unless we add a class to the clipboard icon so I can fade it in only once it has been toggled (since I wouldn't want all clipboards to fade in on page load). Is that possible? It doesn't really bother me as it currently is but I think it could be a tiny bit improved (maybe not "worth it" at this time though).

Reply:
@stacyk My sense is that there is a way to add/remove classes and DOM elements in a way that would allow us to do what we want here, but probably not worth it as part of this PR.